tests/unit/data/observatory/enrichment/test_polygon_enrichment.py

```diff
@@ -567,46 +567,6 @@ def test_enrichment_query_by_polygons_agg_custom(self, dataset_get_mock, _is_ava
 
         assert actual == expected
 
-    @patch('cartoframes.data.observatory.enrichment.enrichment_service._is_available_in_bq')
```

Comment:
I think we should keep this one. If the project is public, the reference to the table name is different (view in private and table in public) https://github.com/CartoDB/cartoframes/blob/develop/cartoframes/data/observatory/enrichment/enrichment_service.py#L137

Probably, we could test in easily, but I thought it was a good idea to validate the full query in a similar way than the rest

Reply:
But we're just validating a string that it's imported from the non-test module, so we're testing nothing, because the other function already test it and the validation string is not in the test
 
https://github.com/CartoDB/cartoframes/pull/1273/files/9a70a64140aebdcfffc972ad2683c3b39f5e17bb#diff-5e7d9427b96cfc9e5c84314e49a69ee9L578

Anyways, not a big deal, feel free to include it again.

Reply:
I have added it again. Let's discuss it after the release